src/logic/operate.js

```diff
@@ -0,0 +1,31 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] Kindly do not make any changes to this file. Please do not disable linters as they help us write cleaner code.

Reply:
Do you have any suggestions for me about how to solve the problem? I have tried everything and I still get the error with the import statement. @Ghiftee . thanks

Reply:
Hi @Reggeditt please try install `big.js` as a dependency then if you use npm for the installation, you can require like this `const Big = require('big.js');` It should fix the error

Reply:
I tried that. It didnt work. And I also checked the dependencies, big.js is listed there so I don't know why the error is showing up in the first place.
When I use const Big = require('big.js');, it gives me the same error.
@Ghiftee

Reply:
Please advice 🙏🏾 @Ghiftee

Reply:
let me show you..